sdk/typescript/introspector/test/testdata/objectParam/index.ts

```diff
@@ -19,7 +23,7 @@ class ObjectParam {
 
   @func()
   upper(msg: Message): Message {
-    msg.content = msg.content.toUpperCase()
+    msg.content = msg.toUpperCase()
```

Comment:
Because the prototype chains aren't respected, this call fails because `toUpperCase` is undefined.

Reply:
It still fails, since the prototype isn't attached.  

You just start with a simple object and just attach the properties onto it
https://github.com/Xiot/dagger/blob/87f82c38ad7aa91bce2d7974dfc098cf363f5f33/sdk/typescript/entrypoint/load.ts#L221

It also looks like having aliases on the input objects messes things up as well, as it does the mapping in the wrong order.  The code that handles the aliasing assumes that its only output types that are aliased.